trlx/trainer/nn/ppo_models.py

```diff
@@ -26,6 +26,10 @@
     whiten,
 )
 
+TMODEL_PROVIDER = Callable[
+    [transformers.PretrainedConfig], transformers.AutoModelForCausalLM
```

Comment:
This won't work for seq2seq, or NeMo once integration there is finished.

Reply:
Will changing to  just `Callable` be enough?

Reply:
Yeah I think so.